Shifty.App/Pages/Statistics.razor

```diff
@@ -0,0 +1,24 @@
+@page "/Statistics"
+@using Components
+@inject NavigationManager NavManager
+
+@if (_user is not null && _user.IsInRole("Board"))
```

Comment:
This type of page seems very verbose for what it does. I think AuthenticatedViews could be used instead, or perhaps the authenticated attribute 🤔

Reply:
I've added #36 for this

Reply:
I think you are right. If there is an easier way to do this, that would be cool! I might need to look at it with you though!